src/js/components/Pagination/PageControl.js

```diff
@@ -4,11 +4,16 @@ import { ThemeContext } from 'styled-components';
 import { Text } from '../Text';
 import { StyledPaginationButton, StyledContainer } from './StyledPageControl';
 
-export const PageControl = ({ control, separator, ...rest }) => {
+export const PageControl = ({
+  control,
+  separator,
+  size: sizeProp,
```

Comment:
Do we have to call this sizeProp? any reason not to just pass it through as size?

Reply:
Yes, it can just be size. Had it differentiated when developing to keep it separate from Button's size.

Reply:
fixed and renamed prop being passed to the StyledPageControl